medical/models/medical_abstract_entity.py

```diff
@@ -195,14 +203,13 @@ def _get_default_image(self, vals):
                     'base', 'static/src/img', 'patient-avatar.png',
                 )
                 with open(img_path, 'r') as image:
-                    base64_image = image.read().encode('base64')
-                    return odoo.tools.image_resize_image_big(base64_image)
+                    return image
 
         Args:
             vals (dict): Values dict as passed to create.
 
         Returns:
-            str: Base64 encoded image if there was one.
+            file: A file-like object representing the image.
             bool: False if error.
             NoneType: None if no result.
         """
```

Comment:
Hrm but this is assuming that the image will always be a local file path. 

One of our PaaS plans is an internally hosted Gravatar for the SSO implementation, which will also end up needing some modules to inject into this default and grab a Gravatar instead. If we switch to use only a local file path here, the image would need to first be downloaded locally then passed to the method. 

While I agree we're duplicating the image open in all the child methods, I think it may be worth it. @tedsalmon any opinions?